pytest_rts/tests/test_e2e.py

```diff
@@ -39,7 +113,11 @@ def test_only_new_functions_are_ran(testdir: Testdir) -> None:
     os.rename(".coverage", COV_FILE)
```

Comment:
It looks like lines above together with `cleanup_caches(str(testdir.tmpdir))` are called in every test and thus are good candidates for [`setup_module()`](https://docs.pytest.org/en/stable/xunit_setup.html) method. Isn't it?

Reply:
I used a pytest fixture to do something like this